x-pack/plugins/fleet/server/services/security/route_required_authz.ts

```diff
@@ -126,6 +126,36 @@ const ROUTE_AUTHZ_REQUIREMENTS = deepFreeze<Record<string, FleetRouteRequiredAut
       },
     },
   },
+
+  // agent status fo policy API
+  [`get:${AGENT_API_ROUTES.STATUS_PATTERN}`]: {
+    any: {
+      fleet: {
+        all: true,
+      },
+      packagePrivileges: {
+        endpoint: {
+          actions: {
+            readPolicyManagement: {
+              executePackageAction: true,
+            },
+            readTrustedApplications: {
+              executePackageAction: true,
+            },
+            readEventFilters: {
+              executePackageAction: true,
+            },
+            readHostIsolationExceptions: {
+              executePackageAction: true,
+            },
+            readBlocklist: {
+              executePackageAction: true,
+            },
```

Comment:
Do we use this API in all of this areas of Security Solution?

I thought it was only used in Policy Details, but maybe I'm wrong. Let me know

Reply:
Yeah, you're right. I'll remove the other artifacts' privileges.